packages/@aws-cdk/cloud-assembly-schema/lib/schema.ts

```diff
@@ -0,0 +1,90 @@
+import * as fs from 'fs';
+import * as jsonschema from 'jsonschema';
+import * as assembly from './manifest';
+
+/**
+ * Protocol utility class.
+ */
+export class Manifest {
+  /**
+   * Save manifest to file.
+   *
+   * @param manifest - manifest.
+   */
+  public static save(manifest: assembly.AssemblyManifest, filePath: string) {
+    fs.writeFileSync(filePath, JSON.stringify(manifest, undefined, 2));
+  }
+
+  /**
+   * Load manifest from file.
+   */
+  public static load(filePath: string): assembly.AssemblyManifest {
+    const raw: assembly.AssemblyManifest = JSON.parse(fs.readFileSync(filePath, 'UTF-8'));
+    Manifest.patchStackTags(raw);
+    Manifest.validate(raw);
+    return raw;
+  }
+
+  /**
+   * Fetch the current schema version number.
+   */
+  public static version(): string {
+    // eslint-disable-next-line @typescript-eslint/no-require-imports
+    return require('../schema/cloud-assembly.metadata.json').version;
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-require-imports
+  private static schema: jsonschema.Schema = require('../schema/cloud-assembly.schema.json');
+
+  private static validate(manifest: assembly.AssemblyManifest) {
+    const validator = new jsonschema.Validator();
+    const result = validator.validate(manifest, Manifest.schema, {
+
+      // does exist but is not in the TypeScript definitions
+      nestedErrors: true,
+
+      allowUnknownAttributes: false
```

Comment:
`true` please (see comment below). We should not be strict about this I think.

Reply:
👍